libkbfs/folder_branch_ops.go

```diff
@@ -3253,6 +3258,18 @@ func (fbo *folderBranchOps) applyMDUpdates(ctx context.Context,
 	return fbo.applyMDUpdatesLocked(ctx, lState, rmds)
 }
 
+func (fbo *folderBranchOps) getLatestRemoteMergedRevision() MetadataRevision {
```

Comment:
If you go with my `headLock` suggestion, you can suffix this function name with "Locked", have it take an `lState`, and then `fbo.headLock.AssertLocked(lState)` here.